src/java/Main.java

```diff
@@ -0,0 +1,49 @@
+package java;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Civil AN=new Civil();
```

Comment:
@Hlieb имена объектов/экземпляров класса всегда с маленькой буквы
```suggestion
        Civil aN=new Civil();
```